lib/git_reflow/git_server/pull_request.rb

```diff
@@ -3,6 +3,24 @@ module GitServer
     class PullRequest
       attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
 
+      APPROVAL_REGEX = /(?i-mx:lgtm|looks good to me|[:\\+1:]|:thumbsup:|:shipit:)/
+
+      def self.minimum_approvals
+        if "#{GitReflow::Config.get('constants.minimum_approvals')}".length > 0
```

Comment:
`GitReflow::Config.get(...)` will return an empty string if it has no key, so there is no need for the check here